todo/forms.py

```diff
@@ -29,8 +33,17 @@ class AddEditTaskForm(ModelForm):
 
     def __init__(self, user, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        
+        # Django's ManyToManyRel is a little odd in that its directional sense 
+        # is not guranteeed and must be tested for. One of these models is User, 
+        # the other is the Groups model (the one group_field points to). We seek 
+        # the attribute with which we can access the set of users in a group.  
+        group_field = getattr(user._meta.model, setting("TODO_USER_GROUP_ATTRIBUTE"), "groups")
+        candidates = (group_field.rel.related_name, group_field.rel.field.attname)
+        user_attr = candidates[1] if group_field.rel.model == user._meta.model else candidates[0]
+        
         task_list = kwargs.get("initial").get("task_list")
-        members = task_list.group.user_set.all()
+        members = getattr(task_list.group, user_attr).all()
```

Comment:
Whew! Funky. Are you confident this will work for any possible alternate group system?

Reply:
Pretty confident. As the only premise is that there is a ManyToMany relation between the group model and `User` (which is fairly safe, but to be honest as `User` is a fairly fixed entity in auth (can be overridden too but I'd guess that unlikely as Django explicitly provides a profile paradigm for extending it which is commonly used) and any new user group model must have a ManyToMany to `User` or a ForeignKey to `User` but either way that is a ToMany relation from User and hence the related_name that an admin must specify in `TODO_USER_GROUP_ATTRIBUTE` is a ToMany relation, hence supporting .all().

To be honest though as this requires setting `TODO_USER_GROUP_ATTRIBUTE` I epxect an admin to be on top of that ;-). To be perfectly safe, you cover the unlikely case that an admin specifies a OneToOne relation between User and their User Groups ... in that instance and that one alone, it warrants a test, to see if `.all()` is implemented to return the one related object (for consistency) or of Django implements another way of getting it.

If the latter, we'd just need to pull the same trick, `getattr` and if `all` exists check if its callable.